tasks/1_golf-score/golf-score.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * GOLF_SCORES - enum, encounteres different nicknames
+ */
+export const GOLF_SCORES = {
+  HOLE_IN_ONE: 'Hole-in-one!',
+  EAGLE: 'Eagle',
+  BIRDIE: 'Birdie',
+  PAR: 'Par',
+  BOGEY: 'Bogey',
+  DOUBLE_BOGEY: 'Double Bogey',
+  GO_HOME: 'Go Home!',
+};
+
+/**
+ * golf-score - returns nickname, depending on
+ * how far above or below par your strokes are.
+ *
+ * @param {Number} par
+ * @param {Number} strokes
+ * @return {String}
+ */
+export function golfScore(par, strokes) {
+  if (strokes === 1) {
```

Comment:
I think switch here is more appropriate.

Reply:
There was switch statement. I replaced it by if statements. 
https://github.com/linkouth/ui-internship-lwo2/pull/1#discussion_r322606566